<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionpsd"/>
    <addaction name="actionLogout"/>
    <addaction name="actionAdvance"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionDelete"/>
    <addaction name="actionDelete_Commodity"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAccount">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/account.png</normaloff>:/images/account.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Account_info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionpsd">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Change_Password.png</normaloff>:/images/Change_Password.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Password_change</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/note_add.png</normaloff>:/images/note_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Delete.png</normaloff>:/images/Delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete-order</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/why_help.png</normaloff>:/images/why_help.png</iconset>
   </property>
   <property name="text">
    <string>help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Search.png</normaloff>:/images/Search.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/dialog_logout.png</normaloff>:/images/dialog_logout.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Logout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Close.png</normaloff>:/images/Close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDelete_Commodity">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Delete.png</normaloff>:/images/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete-Commodity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionAdvance">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/advanced_manager.png</normaloff>:/images/advanced_manager.png</iconset>
   </property>
   <property name="text">
    <string>Advance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
